CenterForOpenScience/osf.io
# [ENG-2652] Update secondary schema

## Purpose

Fixing some issues with the secondary schema.

## Changes

Secondary schema changes

## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify N/A
- Verify

What are the areas of risk?

N/A

Any concerns/considerations/questions that development raised?

I dont love how the schema blocks script works. For the help text, if theres a description, the help text from the schema is completely ignored.

## Documentation

N/A

## Side Effects

N/A
## Ticket

https://openscience.atlassian.net/browse/ENG-2652
